plugin-cgroupv1/tests/oar2.rs

```diff
@@ -0,0 +1,314 @@
+#[cfg(test)]
```

Comment:
There is no need to do this, every file in `tests` is automatically detected as a test :)